### DISCUSSION TITLE
Intel(R) Extension for Scikit-learn 2021.4

### DISCUSSION THREAD

**Discussion by KalyanovD** [intel/scikit-learn-intelex#816] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/intel/scikit-learn-intelex/releases/tag/2021.4.0'>Intel(R) Extension for Scikit-learn 2021.4</a>.</em>

**Comment by FischyM** [comment]
Is there any updates to the issue "[GPU] Examples are failing when run with Visual Studio Solutions on hardware that does not support double precision floating-point operations"? I can't seem to use my Intel UHD Graphics 750 due to this double-precision floating point issue, which is odd since it is fairly new.

**Reply by ethanglaser** [reply]
Hi @FischyM we could not find an issue with that title - are you still facing this issue?

**Reply by FischyM** [reply]
I am not facing this issue anymore.
